docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -166,6 +166,14 @@ For the following steps, change into a clean directory (for example a directory
 These are the only steps which aren't idempotent and have to be completed uninterrupted in one go.
 If you have to recreate the install config or any of the generated manifests you need to rerun all of the subsequent steps.
 
+[NOTE]
+--
+You can add more options to the `install-config.yaml` file. 
+Have a look at the https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example] for more information.
```

Comment:
Should we go for 4.6 already?

Reply:
You have to tell me, current setups are still done in 4.5 as far as I am aware.

Reply:
I have had the same thought but No, not yet. I will change that as part of my current task.